packages/cli/src/commands/rewards/slasher.ts

```diff
@@ -0,0 +1,108 @@
+import { Address } from '@celo/contractkit/lib/base'
+import { ValidatorsWrapper } from '@celo/contractkit/lib/wrappers/Validators'
+import { bitIsSet, parseBlockExtraData } from '@celo/utils/lib/istanbul'
+import { BaseCommand } from '../../base'
+
+export default class Slasher extends BaseCommand {
+  static description = 'Mines for slashable downtime'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  static args = []
+
+  static examples = ['slasher']
+
+  async run() {
+    // const res = this.parse(Slasher)
+    const validators = await this.kit.contracts.getValidators()
+    const downtimeSlasher = await this.kit.contracts.getDowntimeSlasher()
+    const slashableDowntime = await downtimeSlasher.slashableDowntime()
+    let validatorDownSince: number[] = []
+
+    let blockNumber = (await this.kit.web3.eth.getBlockNumber()) - 1
```

Comment:
I'm pretty sure we can subscribe to new block with json_rpc. Maybe that's interesting, idk

Reply:
By default celocli is using HTTP to connect to geth, which doesn't support subscribe.